pnr/nextpnr/fpga_interchange/build.sh

```diff
@@ -36,10 +36,13 @@ mkdir -p $CHIPDB_DIR
 mkdir -p $DEVICES_DIR
 
 # Compute chipdbs for nextpnr-fpga_interchange
+# FIXME: need to change the <device>.device names as soon as more architectures
```

Comment:
@acomodi do you track this in an issue somewhere?

Reply:
Hmm not really, it's not really an issue, probably this should be a TODO rather than a FIXME